### PR TITLE
V5.1 korean update

### PR DESCRIPTION
**Pull request by Jaehyuk-Lee**
testing translations with real situation.

### CODE REVIEW

File: ui/raidboss/data/05-shb/alliance/the_copied_factory.js [hunk 1]
```diff
@@ -103,29 +97,7 @@
...
-            de: 'Tankbuster auf DIR',
-            fr: 'Tankbuster sur VOUS',
-            cn: '死刑点名',
-            ko: '탱버 대상자',
-          };
-        }
-      },
-      infoText: function(data, matches) {
-        if (matches.target == data.me || data.role != 'healer')
-          return;
```
Comment (Jaehyuk-Lee) [inline ui/raidboss/data/05-shb/alliance/the_copied_factory.js]: what are those 
```javascript
      infoText: function(data, matches) {
        if (matches.target == data.me || data.role != 'healer')
          return;
```
was for?
my brain cannot logically understand this.
if healer, does not send message?

Comment (quisquous) [reply ui/raidboss/data/05-shb/alliance/the_copied_factory.js]: I think this is a bad copy and paste.  I am not sure either.

File: ui/raidboss/data/05-shb/alliance/the_copied_factory.js [hunk 2]
```diff
@@ -283,13 +243,7 @@
...
         return data.me == matches.target;
       },
-      alertText: {
-        en: 'Spread',
-        de: 'Verteilen',
-        fr: 'Dispertion',
-        cn: '分散',
-        ko: '산개',
-      },
+      response: Responses.spread(),
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/alliance/the_copied_factory.js]: Responses.spread('alert')

File: ui/raidboss/data/05-shb/alliance/the_copied_factory.js [hunk 3]
```diff
@@ -401,27 +349,15 @@
...
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/alliance/the_copied_factory.js]: Responses.spread('alert')

File: ui/raidboss/data/05-shb/alliance/the_copied_factory.js [hunk 4]
```diff
@@ -504,13 +440,7 @@
...
         return data.me == matches.target;
       },
-      infoText: {
-        en: 'Tank Buster on YOU',
-        de: 'Tankbuster auf DIR',
-        fr: 'Tankbuster sur VOUS',
-        cn: '死刑点名',
-        ko: '탱버 대상자',
-      },
+      response: Responses.tankBuster(),
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/alliance/the_copied_factory.js]: I guess the condition makes this only on you.  This should be Responses.tankBuster('info') though to match previous text? I don't think this is super serious.

File: ui/raidboss/data/05-shb/alliance/the_copied_factory.js [hunk 5]
```diff
@@ -874,23 +759,20 @@
         'Marx [LR]': '(Linker|Rechter) Marx',
         'Medium Exploder': 'mittelgroße Selbstzerstörung',
         'Multi-leg Medium Model': 'mittelgroßes mehrbeiniges Modell',
-        'Quality assurance will be sealed off': 'Noch 15 Sekunden, bis sich die Warenkontrollhalle schließt',
+        'Quality assurance': 'Warenkontrollhalle',
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/alliance/the_copied_factory.js]: Thanks! I thought I fixed these, but maybe I missed them.

File: ui/raidboss/data/05-shb/alliance/the_copied_factory.js [hunk 6]
```diff
@@ -729,8 +620,8 @@
       },
       alertText: {
         en: 'Laser Buster on YOU',
-        de: 'Laser Tankbuster auf DIR',
-        fr: 'Laser Tankbuster sur VOUS',
+        de: 'Laser tankBuster auf DIR',
```
[the remark this answers is not in the document]
Comment (Jaehyuk-Lee) [reply ui/raidboss/data/05-shb/alliance/the_copied_factory.js]: I did some wrong copy&paste on this file maybe.. git went wrong somehow.

File: ui/raidboss/data/05-shb/alliance/the_copied_factory.js (hunk 5 above)
Comment (Jaehyuk-Lee) [reply ui/raidboss/data/05-shb/alliance/the_copied_factory.js]: No this was fixed by you, but I did something wrong, git went wrong somehow.

Comment (Jaehyuk-Lee) [reply ui/raidboss/data/05-shb/alliance/the_copied_factory.js]: This branch is forked from maybe two weeks ago? so those changes are not adopted yet here.

**Comment by quisquous** [comment]
Is this still a draft? Should I merge it?

**Comment by Jaehyuk-Lee** [comment]
@quisquous I'm working on 5.1 dungeon. others are done

**Comment by Jaehyuk-Lee** [comment]
I have added more commit, but this PR is not showing them now. Probably due to Github server issue.
https://github.com/Jaehyuk-Lee/cactbot/commits/v5.1-Korean
More commits there.

**Pull request merged**